bundle/Form/Data/Content/ContentUpdateAlwaysAvailableData.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\BetterIbexaAdminUIBundle\Form\Data\Content;
+
+use Ibexa\Contracts\Core\Repository\Values\Content\ContentInfo;
+
+class ContentUpdateAlwaysAvailableData
+{
+    private ?ContentInfo $contentInfo;
+    private ?bool $alwaysAvailable;
+
+    public function __construct(?ContentInfo $contentInfo = null)
```

Comment:
Why does the constructor even allow `null`, since you don't create this class in `createContentAlwaysAvailableUpdateForm` if you don't have a proper content info object?

Reply:
Same as above applies.